### PR TITLE
Add ACTUAL quantized add for quickstart support

### PR DESCRIPTION
**Pull request by mbartling**

### CODE REVIEW

**Comment by mbartling** [comment]
Rebased

**Comment by dboyliao** [comment]
screen shot 2018-04-12 at 9 59 26 pm

I generate the pb file with your script and test it against `utensor-cli`, it give me this error msg.
It seems like there is an extra `^` added to the node name.
That's why the `KeyError` pop out.

**Comment by mbartling** [comment]
Have you tried running it with `--output-nodes z_1`. I ran into this issue with the freeze constants and transform graph stuff.

**Comment by dboyliao** [comment]
Yes, it works with `--output-nodes`.
I'd like to find out why.

File: tests/quantized_add/generate_pb.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        x = tf.constant(np.array([1, 2, 3, 4]), dtype=tf.float32, name="x")
+        y = tf.constant(np.array([1, 1, 1, 1]), dtype=tf.float32, name="y")
+        z = tf.add(x, y, name="z")
+    with tf.Session(graph=graph) as sess:
+        # Force the quantization
+        transformed_graph_def = TransformGraph(sess.graph_def, [],
+                                               ["z"], ["quantize_weights", "quantize_nodes"])
+        
+        # Load the transformed graph into current context
+        tf.import_graph_def(transformed_graph_def, name="")
```
[the remark this answers is not in the document]
Comment (mbartling) [reply tests/quantized_add/generate_pb.py]: cool! I am a total dumbass. will try that now.

[the remark this answers is not in the document]
Comment (mbartling) [reply tests/quantized_add/generate_pb.py]: @dboyliao Do you know an easy way to evaluate z on the new graph?

[the remark this answers is not in the document]
Comment (dboyliao) [reply tests/quantized_add/generate_pb.py]: No.
But I will evaluate it in the old graph session.
It's just a reference output.
Doesn't matter evaluate it in the old or new graph.

**Comment by dboyliao** [comment]
Looking good.
Merge!

**Comment by mbartling** [comment]
swag swag

**Pull request merged**